runtime/src/bank/tests.rs

```diff
@@ -8013,44 +8013,208 @@ fn test_compute_active_feature_set() {
     assert!(bank.feature_set.is_active(&test_feature));
 }
 
+fn set_up_account_with_bank(
+    bank: &mut Bank,
+    pubkey: &Pubkey,
+    lamports: u64,
+    data: Vec<u8>,
+) -> AccountSharedData {
+    let new_account = AccountSharedData::from(Account {
+        lamports,
+        data,
+        ..Account::default()
+    });
```

Comment:
It might be good to also take the owner and executable flag into account for a full test, since those will change for the feature gate program replacement.